### PR TITLE
Added support for Expeditor

### LINKED ISSUE #126: Setup this repository to be released via Expeditor
We should wire this up for expeditor so we get automated changelogs and releases via chatops.

### PR DESCRIPTION
**Pull request by vijaymmali1990**
- Added CHANGELOG.md which contains all merged PRs.
- Added VERSION file.
- Added update_version.sh file containing script.
- Added config.yml file.

Signed-off-by: vijaymmali1990 <vijay.mali@msystechnologies.com>

### Description
Added a support to this repository so that it can be released via Expeditor

### Issues Resolved
Fixes #126

### CODE REVIEW

**Comment by dheerajd-msys** [comment]
@btm, @tas50 Could you please take a look into it. Thanks.

File: .expeditor/update_version.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# After a PR merge, Chef Expeditor will bump the PATCH version in the VERSION file.
+# It then executes this file to update any other files/components with that new version.
+#
+
+set -evx
+
+sed -i -r "s/^(\s*)VERSION = \".+\"/\1VERSION = \"$(cat VERSION)\"/" lib/knife_ec_backup/version.rb
```
Comment (btm) [inline .expeditor/update_version.sh]: `lib/knife_ec_backup_version.rb` has single quotes around the version, so you need this instead:

```
sed -i -r "s/^(\s*)VERSION = '.+'/\1VERSION = '$(cat VERSION)'/" lib/knife_ec_backup/version.rb
```

Comment (btm) [reply .expeditor/update_version.sh]: (chefstyle wants double quotes around version, but let's not worry about making chefstyle happy in this PR.)

Comment (btm) [reply .expeditor/update_version.sh]: might as well put a comment in `lib/knife_ec_backup/version.rb` that says when you do change these to double quotes to also update `.expeditor/update_version.sh`.

File: VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+2.4.1
```
Comment (btm) [inline VERSION]: weird that 2.4.0 was our last release to rubygems, but 2.2.0 is our last tag. that's fine, nothing to change here. just pointing out this is correct despite the tags.

**Pull request merged**